content/en/monitors/notify/variables.md

```diff
@@ -303,6 +202,163 @@ This is the escalation message @dev-team@company.com
 {{% /tab %}}
 {{< /tabs >}}
 
+
+## Notification message enrichment
+
+### Attribute and tag variables
+
+#### Multi-alert variables
+
+Multi-alert variables can be used in [multi-alert monitors][1] based on the dimension selected in the multi-alert group box.
+Enrich notification to dynamically include the value associated with the group by dimension in each alert.
+
+##### Query group by tag
```

Comment:
Let's do two tabs: tags and attributes instead of two sections. 

As the two is the exact same behaviour, the only difference is that for attribute a @ is required. However the @ is included in the query path so in a way it's the same thing but is worth calling out. 

WDYT about using tabs instead?